.gitignore

```diff
@@ -0,0 +1,122 @@
+
+# Created by https://www.gitignore.io/api/python,django
```

Comment:
`.ignore` *DESCRIPTION*
This file is used by git in order to understand which file to ignore and not keep track off, you can take a look at the official docs here: https://git-scm.com/docs/gitignore

Why do we want to not track all the files?
- Some files are auto generated, and therefore tracking changes in those is futile as all of us will have them auto generated locally on our machine. These will simply create noise in every PR and don't provide value
- Some files are not useful and get generated occasionally by different processes, for example `.log`
- Some files may contain confidential information. For example the database is too big & may contain confidential information. Each machine should have it's own database for development purposes, we will populate the data as we go along and all have different data in it. For production purposes we will have a shared DB that will be hosted somewhere like Heroku and protected from exposing sensitive data.

**Bonus points**
Google the folders and files below you don't recognise to see what they are and why we mights be ignoring them from being tracked.